android/app/src/main/res/values/color.xml

```diff
@@ -2,4 +2,5 @@
 <resources>
     <color name="primaryColor">#ffffff</color>
     <color name="primaryColorDark">#d7ccc8</color>
+    <color name="splashscreen_background">#51c2af</color>
```

Comment:
nit: follow surrounding capitalization convention, which is camelCase

Reply:
(If using that Expo package, it apparently relies on the specific resource names. That's not a great API choice, and it's not real clearly stated in those setup instructions, but it's e.g. here in the implementation:
https://github.com/expo/expo/blob/227455e3d6f4bc9d05ca84109db5cbc03a56e342/packages/expo-splash-screen/android/src/main/java/expo/modules/splashscreen/NativeResourcesBasedSplashScreenViewProvider.kt#L32-L36
But we're not, so we don't.)

Reply:
... Oh I see, using the Expo package is the next commit. :slightly_smiling_face:

Reply:
Through trial and error, I noticed that it did depend on it literally being called "splashscreen_background" and thought it was odd, but I didn't locate it in the Expo package.

My preference was actually something like "brand"—does that sound right? It would be nice to have something reusable on the Android native side for the brand color. And we may be able to bring it over the bridge with RN v0.63.0's fancy new [`PlatformColor` API](https://reactnative.dev/docs/platformcolor).

Reply:
> My preference was actually something like "brand"—does that sound right? It would be nice to have something reusable on the Android native side for the brand color. And we may be able to bring it over the bridge with RN v0.63.0's fancy new [`PlatformColor` API](https://reactnative.dev/docs/platformcolor).

Sure, SGTM!